build/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ option(TRTIS_ENABLE_CUSTOM "Include Custom backend in server" OFF)
 option(TRTIS_ENABLE_HTTP "Include HTTP API in server" ON)
 option(TRTIS_ENABLE_GRPC "Include GRPC API in server" ON)
 option(TRTIS_ENABLE_HTTP_V2 "Enable version 2 of the HTTP API in server" OFF)
+option(TRTIS_ENABLE_HTTP_H2O "Enable h2o version of the HTTP API in server" OFF)
```

Comment:
Since you are doing this on a separate branch you should just make all the changes to replace libevhtp with h2o. So don't add a build flag, remove libevhtp from make, etc.

Reply:
Since support for h2o is not complete I have kept the older libevhtp server for now. I will phase it out once all h2o functionalities are added. Yes, I will also remove the build flag in the future and remove libevhtp from make.

Reply:
Done